src/NuGet.Clients/VsExtension/NuGetPackage.cs

```diff
@@ -1091,8 +1094,11 @@ private void BeforeQueryStatusForPackageRestore(object sender, EventArgs args)
                 // Enable the 'Restore NuGet Packages' dialog menu
                 // a) if the console is NOT busy executing a command, AND
                 // b) if the solution exists and not debugging and not building AND
-                // c) if there are no NuGetProjects. This means that there no loaded, supported projects
-                command.Enabled = !ConsoleStatus.IsBusy && IsSolutionExistsAndNotDebuggingAndNotBuilding() && SolutionManager.GetNuGetProjects().Any();
+                // c) if the solution is DPL enabled or there are NuGetProjects. This means that there loaded, supported projects
+                // TODO: checking for DPL more is a temporary heck until we've the capability to get nuget projects 
+                // even in DPL mode
+                command.Enabled = !ConsoleStatus.IsBusy && IsSolutionExistsAndNotDebuggingAndNotBuilding() &&
```

Comment:
Spoke offline about this. We agreed is make more sense to keep these separate for now.